packages/server/src/services/agent.service.ts

```
@@ -105,20 +105,22 @@ export class AgentService implements IAgentService {
 
   async *handleUserRequestWebsocket(
     agent: any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)